en/docs/learn/adding-users-and-roles.md

```diff
@@ -1,11 +1,16 @@
 # Adding Users and Roles
 
-## Scenario
-This tutorial demonstrates user and role management in WSO2 Identity Server. User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
+User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
 
 This tutorial demonstrates how to create a new user, create a role with a set of permissions, and then log in to the dashboard as the newly created user. 
```

Comment:
WSO2 Identity Server Dashboard with the newly created user's credentials.

Reply:
Fixed 
485f165